ci/docker/linux-apt-r.dockerfile

```diff
@@ -43,23 +46,31 @@ RUN apt-get update -y && \
         clang-tidy \
         # R CMD CHECK --as-cran needs pdflatex to build the package manual
         texlive-latex-base && \
+    # Need locales so we can set UTF-8
+    apt-get install -y locales && \
```

Comment:
How about putting this into the above `apt-get install` instead of adding a new `apt-get install`?

Reply:
I made it separate because the code comments implied that all that was copy-pasted from somewhere else, and I didn't want someone in the future to accidentally undo it, but sure, I can move it.